Resources/public/js/views/actions/ez-translateactionview.js

```diff
@@ -45,7 +45,9 @@ YUI.add('ez-translateactionview', function (Y) {
          */
         render: function () {
             var container = this.get('container'),
-                translationsList = this.get('content').get('currentVersion').getTranslationsList(),
+                content = this.get('content'),
+                translationsList = content.get('currentVersion').getTranslationsList(),
+                mainLanguageCode = content.get('mainLanguageCode'),
```

Comment:
those changes also can be reverted as you don't need `mainLanguageCode`

Reply:
You are right, fixed.